packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts

```diff
@@ -17,7 +17,7 @@ export interface TelemetryEventMessageData {
 	/**
 	 * Contents of the telemetry event
 	 */
-	contents: ITelemetryBaseEvent;
+	contents: ITelemetryBaseEvent[];
```

Comment:
Let's update the docs of this class to point out it's also used by the new message.